### ISSUE TITLE
mapped organs versus homologous organs

### ISSUE THREAD

**Issue by ANiknejad** [BgeeDB/anatomical-similarity-annotations#9]
Hi Chris,

here some examples of multiple organs not related by homology links, instead just mapped together

UBERON:0007798, UBERON:0002049  
vascular system, vasculature 

PORO:0000018, UBERON:0000165  
osculum, mouth

UBERON:0006566, UBERON:0003382  
left ventricle myocardium, cardiac muscle of left ventricle

...

how to manage that?
- by merging in Uberon? (or creating a parent terms?)
- by mapping in our homology file?

Here an example of 'true homology' between 2 organs, but one from an species-specific ontology:
UBERON:0001941, ZFA:0000302 
lateral habenular nucleus, ventral habenular nucleus (zebrafish)

---> maybe needs of an Uberon class here?
please see
https://github.com/obophenotype/uberon/issues/542

An example of 'true homology' between 2 organs, both from species-specific ontologies:

FBbt:00007251, CL:0000740  
rhabdomeric photoreceptor cell (Drosophila), retinal ganglion cell

Sorry to give you all these examples in bulk and many thanks in advance for your comments,

Cheers,

Anne

**Comment by fbastian** [comment]
cc @cmungall

**Comment by cmungall** [comment]
> osculum, mouth

Should not be merged. I'm not sure what you mean by "just mapped together". Currently the only linkage between these is your single low-confidence homology assertion in the bgee homology table. I think we're good with this situation, yep?

> habenular nuclei

the ZFA classes will be subclasses of the generic 'habenular nucleus' on the next release. We could merge these into lateral & medial or leave them distinct to be grouped by homology assertions, up to you.

> vascular system, vasculature

Seems like a merge candidate. As an aside, ncit defines the vascular system to be "The cardiovascular and lymphatic systems, collectively", but this is inconsistent with their placement as a part of the cardiovascular system

> left ventricle myocardium, cardiac muscle of left ventricle

ugh. I need to review notes & tickets for myocardium vs cardiac muscle tissue, there are some oddities here. but either way, I'm not sure of the need to precompose such fine grained distinctions at the lateral level.

When should homology statements for lateral structures be asserted? Note I don't mean lateral homology, between two halves of one organism, I mean between corresponding lateral structures in two species, e.g. my left ventricle and a mouse left ventricle. For any symmetric body part we can just infer this. But this may not be safe in all cases, e.g. heart.

**Comment by ANiknejad** [comment]
"
 cmungall commented 2 days ago

```
osculum, mouth
```

Should not be merged. I'm not sure what you mean by "just mapped together". Currently the only linkage between these is your single low-confidence homology assertion in the bgee homology table. I think we're good with this situation, yep?
"

...well...no more clear even for me...!!!! 

"just mapped together"
I guess, I wanted to express that sometimes I do not know which class to select in order to report the homology assertion as we do (and well do ;-) ), sometimes it is difficult to choose the correct term to report the homology, it is not that the class is missing, it is just difficult to know what the authors are talking about...and to match with the class definition already available...

**Comment by cmungall** [comment]
Yes, and this may be especially difficult when we have a loose concept represented by multiple more formally defined classes. If the osculum is homologous to the mouth, what is the mouth exactly? 

which of these?
- [UBERON:0000165](http://purl.obolibrary.org/obo/UBERON_0000165) ! mouth
- [UBERON:0000166](http://purl.obolibrary.org/obo/UBERON_0000166) ! oral opening
- [UBERON:0000167](http://purl.obolibrary.org/obo/UBERON_0000167) ! oral cavity

Maybe it would be best to draw up some guidelines. E.g. for gene expression curation you never want to choose a space. Genes are always initially expressed in a cell, which may be located in a space, but not part of it. You can just ditch all subclasses for space for all bgee curation.

Then there is the shadow hierarchies: e.g. bones and skeletons; we should get the inference rules such that it doesn't matter but even then it would be useful to have some curation guidelines.

Is this the sort of thing you're getting at?

**Comment by ANiknejad** [comment]
Hi Chris,
...very late...

> Is this the sort of thing you're getting at?

yes, exactly and thank you for the advice about how to select the right term, depending on the type of evidence, and about needs to create some new guidelines.
Cheers,
Anne
